UnitTests/Text/AutocompleteTests.cs

```diff
@@ -153,7 +153,7 @@ public void KeyBindings_Command ()
 			Assert.Equal (3, g.AllSuggestions.Count);
 			Assert.True (tv.NewKeyDownEvent (new (tv.Autocomplete.SelectionKey)));
 			tv.PositionCursor ();
-			Assert.Equal ($"fortunately Fortunately super feature.", tv.Text);
+			Assert.Equal ($"Fortunately Fortunately super feature.", tv.Text);
```

Comment:
With this change I'm seeing that this issue already existed before the Key class implementation. Good catch.